parm/post_avblflds.xml

```diff
@@ -6569,6 +6573,76 @@
        <scale>4.0</scale>
     </param>
 
+    <param>
+       <post_avblfldidx>912</post_avblfldidx>
```

Comment:
UPP ID 912 was assigned for REFD_on_isothermal. You may see post_avblflds_raphrrr.xml and fv3sar_post_avblflds.xml.
Can you sign SBTAHI7 as new ID?

Reply:
@WenMeng-NOAA - We updated the set of Himawari fields to be 969-978 to avoid the conflict with raphrrr and fv3sar.  The post_avblflds.xml file and related source code (RQSTFLD.F and CALRAD_WCLOUD_newcrtm.f) have been updated according and have been pushed to this PR.  @hertneky is going to test on our end to make sure this is all good.

Reply:
@hertneky Good catch.  I added the change and pushed to the branch.

Reply:
@fossell One more fix needed that propagated from when I added the grib2 capabilities for Himawari8. In CALRAD_WCLOUD_newcrtm.f L2130, change to igot=iget(968+ichan) so CH7 is output. Thanks for catching that!
With this last change - test run is successful!

Reply:
@hertneky - Thanks!  Fixed and push to the branch.
@WenMeng-NOAA - I think this issue with ID for SBTAHI7 is resolved.

Reply:
Thanks!